app/assets/javascripts/snowcrash/modules/index_table.js.coffee

```diff
@@ -0,0 +1,188 @@
+class IndexTable
+  project: null
+  selectedColumns: []
+  selectedItemsSelector: ''
+  columnIndices: {}
+
+  constructor: (@itemName) ->
+    @$jsTable     = $('.js-index-table')
```

Comment:
let's start moving away from `js-` classes and use data-behaviour for JS behaviour

https://signalvnoise.com/posts/3167-code-spelunking-in-the-all-new-basecamp

Maybe split this in a separate card?

Reply:
Yes please